### ISSUE TITLE
Top containers don't have instances, so I ran into issues with the update to the 'find_container_instances' method, and we wanted to have custom mappings for our top containers... phew :)

### ISSUE THREAD

**Issue by fordmadox** [AtlasSystems/ArchivesSpace-Aeon-Fulfillment-Plugin#36]
To get around that issue for the time being, I added the following:  https://github.com/YaleArchivesSpace/aspace-deployment/commit/a92d98ec1712dee238e4c42dbeb3e4bdee189f3b. 

Okay to add something like to the core branch?  If so, should I submit a pull request, or is there a better way to handle that?

**Comment by stokesd** [comment]
Absolutely, a pull request for this would be welcome and appreciated!  Thanks!
